serverless/README.md

```diff
@@ -70,8 +70,15 @@ You can configure the library by add the following parameters:
 
 ```yaml
 # Whether to add the Lambda Layers, or expect the user to bring their own. Defaults to true.
+# If this is true, 'pythonLibraryVersion' is required if any Lambda function has a Python runtime, and 'nodeLibraryVersion' is required if any Lambda function has a Node.js runtime.
 addLayers: true
 
+# The Python Lambda Library version to install to the Python Lambda functions. This is required if 'addLayers' is true and you are deploying one or more Python Lambda functions. The latest version is 19.
+pythonLibraryVersion: ""
+
+# The Node.js Lambda Library version to install to the Node.js Lambda functions. This is required if 'addLayers' is true and you are deploying one or more Node.js Lambda functions. The latest version is 25.
+nodeLibraryVersion: ""
```

Comment:
Since the previous param is `addLayers`, would it be clearer if these were named `pythonLayerVersion` and `nodeLayerVersion`? Unsure of the language here since we want to move from 'Lambda layers' to 'Lambda Library' overall, but this could be confusing if we still have the `addLayers` param.

Reply:
Yeah, it should be "xxxLayerVersion" here. We are referring it as library now because it can be distributed through layer or other package managers, but here you are specifically adding the library through layer, "layer" makes most sense.